packages/theme/css/characteristics/separable-next.css

```diff
@@ -1,9 +1,9 @@
 .salt-theme.salt-theme-next {
   --salt-separable-borderStyle: solid;
 
-  --salt-separable-primary-borderColor: var(--salt-palette-alpha-contrast-mediumLow);
-  --salt-separable-secondary-borderColor: var(--salt-palette-alpha-contrast-low);
-  --salt-separable-tertiary-borderColor: var(--salt-palette-alpha-contrast-lower);
+  --salt-separable-primary-borderColor: var(--salt-palette-alpha-contrast-mediumHigh);
+  --salt-separable-secondary-borderColor: var(--salt-palette-alpha-contrast-medium);
+  --salt-separable-tertiary-borderColor: var(--salt-palette-alpha-contrast-mediumLow);
```

Comment:
`--salt-separable-background-hover` is referencing a deprecated token. It should be pointing to `--contrast-lower` and therefore changing to `--contrast-mediumLow`